R/CohortConstruction.R

```diff
@@ -145,16 +132,22 @@ getInclusionStatisticsFromFiles <- function(cohortIds = NULL,
         summaryStats = filter(summaryStats, .data$cohortDefinitionId == cohortId),
         simplify = simplify
       )
-    cohortResult$cohortDefinitionId <- cohortId
-    result <- dplyr::bind_rows(result, cohortResult)
+    if (!is.null(cohortResult)) {
```

Comment:
I'm not sure how to add it to this PR but this code is never run in our tests so cohortResult is always null when we run it here.

Reply:
Don't worry about it in this PR, I will add an issue at some point to create some unit tests to fully tests this (probably with some mock exported data).